Bugzilla/Search.pm

```diff
@@ -623,6 +625,7 @@ sub COLUMNS {
 
     'attachments.count'  => 'COUNT(DISTINCT map_attachments_count.attach_id)',
```

Comment:
Any idea why it's `XXX.count` sometimes but `XXX_count` other times?

Reply:
Inconsistency is the best guess. I prefer attachments_count instead of attachments.count since the latter looks like it is referring to a column in the attachments table instead of an identifier of a special search term. But we won't go back and change all of them to match unless we need to for now. Technically they both work the same under the hood.